src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml

```diff
@@ -22,11 +27,63 @@ KDDW.TitleBarBase {
     implicitHeight: gripButton.implicitHeight
     heightWhenVisible: expectedHeight
 
+    Component.onCompleted: {
+        if (effectsPanelWidth == 0) {
+            console.warn("effectsPanelWidth is not set ; doing some guesswork")
+            effectsPanelWidth = 240
+        }
+    }
+
+    onShowEffectsPanelChanged: {
+        rowLayout.effectsTitleBar.visible = showEffectsPanel
```

Comment:
you can use effectsTitleBar directly 
or even better make an alias for root
`property alias showEffectsPanel: effectsTitleBar.visible`